From a0feed301acae30c28a64fadd1e1763f0efa8256 Mon Sep 17 00:00:00 2001
From: Johannes Lorenz <johannes89@ist-einmalig.de>
Date: Sun, 9 Oct 2016 11:06:23 +0200
Subject: [PATCH 1053/1936] Fixed compile errors from
 0ef9e954fe36e62538c73b028922087223a25fdd . Removed warnings caused by
 inconsistent use of the override keyword.

---
 src/CMakeLists.txt        | 1 +
 src/Plugin/AbstractFX.hpp | 4 +++-
 2 files changed, 4 insertions(+), 1 deletion(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index c2759752..2f3a8edf 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -260,6 +260,7 @@ endif()
 add_definitions(
 	 -Wall
 	 -Wextra
+	 -Wno-inconsistent-missing-override
 	 )
 if(NOT AVOID_ASM)
 	message(STATUS "Compiling with x86 opcode support")
diff --git a/src/Plugin/AbstractFX.hpp b/src/Plugin/AbstractFX.hpp
index 254a20d5..2bcb2726 100644
--- a/src/Plugin/AbstractFX.hpp
+++ b/src/Plugin/AbstractFX.hpp
@@ -97,7 +97,9 @@ protected:
     */
     uint32_t getVersion() const noexcept override
     {
-    return d_version(version.major(), version.minor(), version.revision());
+        return d_version(version.get_major(),
+                         version.get_minor(),
+                         version.get_revision());
     }
 
    /* --------------------------------------------------------------------------------------------------------
-- 
2.47.0

